site/tutorials/snippets/11/install/configureIstio.md

```diff
@@ -44,13 +44,34 @@ spec:
   - host: <IP-ADDRESS>.nip.io
     http:
       paths:
-      - pathType: Prefix
-        path: /
+      - path: /
+        pathType: Prefix
         backend:
           service:
             name: api-gateway-nginx
             port:
               number: 80
+```
+
+Please be aware, when using OpenShift 3.11, instead using the above manifest, use the following one, as it uses an already deprecated apiVersion.
```

Comment:
```suggestion
Please be aware, when using OpenShift 3.11, instead of using the above manifest, use the following one, as it uses an already deprecated apiVersion.
```